tasks/hyrax-metadata-updates/index.js

```diff
@@ -166,7 +166,7 @@ function addHyraxUrlToUmmG(metadata, hyraxUrl) {
   }
   const url = {
     URL: hyraxUrl,
-    Type: 'GET DATA',
+    Type: 'USE SERVICE API',
```

Comment:
Do we need to note this in the CHANGELOG?

Reply:
Added.  I thought only tests are affected.